index.bs

```diff
@@ -592,7 +592,6 @@ A [=sensor=] has an associated <dfn>current polling frequency</dfn> which is ini
 <pre class="idl">
 [SecureContext]
 interface Sensor : EventTarget {
-  readonly attribute SensorState state;
```

Comment:
I am fine with exposing the activated attribute. Then we can also see whether people use it or not or need something more powerful